debian/copyright

```diff
@@ -3,21 +3,23 @@ Upstream-Name: sosreport
 Upstream-Contact: Jake Hunsaker <jacob.r.hunsaker@gmail.com>
 Source: https://github.com/sosreport/sos
 
-License: GPL-2+
- This program is free software; you can redistribute it and/or modify
- it under the terms of the GNU General Public License as published by
- the Free Software Foundation; either version 2 of the License, or (at
- your option) any later version.
+Files: *
+Copyright: (C) 2012-2017 Bryn M. Reeves
+  (C) 2007-2024 Red Hat, Inc.
+  (C) 2012-2024 Canonical Ltd.
```

Comment:
This is based on the work we have already done in Debian packaging, so just mimicking here. The Debian Developers is what made this change, to reflect the contributions.